optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -17,18 +17,17 @@ You will build a REST application that optimizes a school timetable for students
 image::QuickStart/SpringBoot/timeTableAppScreenshot.png[]
 
 Your service will assign `Lesson` instances to `Timeslot` and `Room` instances automatically
-by using AI to adhere to hard and soft scheduling _constraints_, such as:
+by using AI to adhere to hard and soft scheduling _constraints_, such as the following examples:
 
 * A room can have at most one lesson at the same time.
+//consider `one lesson at a time`
```

Comment:
The same was to emphasis that, but I am ok with the change too. Your call.

Reply:
That's OK. Keep it.